docs/services/shippingservice.md

```diff
@@ -0,0 +1,146 @@
+# Shipping Service
+
+This service is responsible for providing shipping information including pricing
+and tracking information, when requested from Checkout Service.
+
+Shipping service is built primarily with Tonic, Reqwest, and opentelemtry. Other
```

Comment:
opentelemtry -> OpenTelemetry libraries/components?

Reply:
fixed in commit